catalog/internals/webpack/webpack.prod.js

```diff
@@ -6,7 +6,6 @@ const TerserPlugin = require('terser-webpack-plugin')
 module.exports = require('./webpack.base')({
   mode: 'production',
 
-  // In production, we skip all hot-reloading stuff
   entry: {
```

Comment:
if this is the same in both prod and dev, we can move it to the base

Reply:
It's different for desktop